src/Doctrine/Instantiator/Exception/UnexpectedValueException.php

```diff
@@ -31,12 +32,12 @@
 class UnexpectedValueException extends BaseUnexpectedValueException implements ExceptionInterface
 {
     /**
-     * @param ReflectionClass $reflectionClass
-     * @param Exception       $exception
+     * @param ReflectionClass     $reflectionClass
+     * @param Exception|Throwable $exception
      *
      * @return self
      */
-    public static function fromSerializationTriggeredException(ReflectionClass $reflectionClass, Exception $exception)
```

Comment:
Yeh, I agree it's unlikely. I just thought I'd point it out as a potential flag.